<launch>

	<node pkg="final" type="PathPlanner.py" name="path_planner" output="screen" >
	</node>

</launch>
